### PR TITLE
Selection route

### PR DESCRIPTION
**Pull request by EdithBenvenuto**
I made the nav to single drug page on selection, sending the id parameter:
Captura de pantalla (4)
Captura

### CODE REVIEW

**Review by alival000 — APPROVED** [review]
Looks good to me!

File: src/main/react-webapp/src/App.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```
Comment (mcollins42) [inline src/main/react-webapp/src/App.js]: Is this import used in this file?

File: src/main/react-webapp/src/components/Home.js [hunk 2]
```diff
@@ -1,12 +1,36 @@
...
 
 const Home = () => {
 
+    let history = useHistory();
+
+    const handleChange = e => {
+        console.log(e.target.value, e.target.selectedOptions[0].label)
+        const value = e.target.value
+        const label = e.target.selectedOptions[0].label
+        console.log(e.target)
```
Comment (mcollins42) [inline src/main/react-webapp/src/components/Home.js]: probably want to remove console.log and commented out lines if this is now working.

**Review by mcollins42 — APPROVED** [review]

**Comment by mcollins42** [comment]
I would also remove .Home.js.swp from this PR. We should probably add swap files to .gitignore.

**Pull request merged**